tests/bootstrap.php

```diff
@@ -82,6 +82,10 @@ public function install_wc() {
 
 		WC_Install::install();
 
+		// Initialize the WC API extensions.
+		\Automattic\WooCommerce\Admin\Install::create_tables();
+		\Automattic\WooCommerce\Admin\Install::create_events();
+
```

Comment:
@claudiosanches I needed to add this in order to bootstrap wc-admin tables for tests. It seems like probably not the best place, do you have a suggestion here?

Reply:
Not sure what we can do.
@rodrigoprimo can you help us?

Reply:
IMO we can leave it as is for now. I agree that is not great but I don't think we currently have an alternative and I don't think it is worth creating something now.